src/Resources/Resources/ChangeLog.md

```diff
@@ -19,6 +19,7 @@
 -->
 
 ## Upcoming Release
+* Added if condition to verify the returned Provider object [#16370]
```

Comment:
Change log is supposed to be read by our customers/users so I think we should hide impelemntation/design details like "if condition" but focus more on the behavior
```suggestion
* Fixed an issue where `Get-AzPolicyAlias` errors out when .....[#16370]
```